packages/server/routes/api/users.js

```diff
@@ -89,7 +92,55 @@ router.get(
         message: error.message,
       });
     }
-  }
+  },
+);
+
+router.put(
+  "/:userId/new",
+  AuthController.authenticate,
+  userIdValidator,
+  updateNewUserValidator,
+  async (request, response) => {
+    const result = validationResult(request);
+
+    if (!result.isEmpty()) {
+      return response.status(400).json({
+        status: "fail",
+        data: result.array(),
+      });
+    }
+    const validatedData = matchedData(request);
+    const { userId } = matchedData(request);
+
+    try {
+      // check if user is new
+      const user = await UserController.getUser(userId);
+      if (user.isNew === false) {
+        return response.status(400).json({
+          status: "fail",
+          data: {
+            userId: "User is not new!",
+          },
+        });
+      }
```

Comment:
This is part of the validation process and should be moved to an express-validator instead of here in the route handler function.